spec/client_spec.rb

```diff
@@ -43,6 +43,13 @@ def loop_times(times)
     @client.update_users(@random_users)
   end
 
+  it 'properly sets up a new client' do
+    client = StreamChat::Client.new(ENV['STREAM_KEY'], ENV['STREAM_SECRET'], base_url: ENV['STREAM_CHAT_URL'])
+    client.set_http_client(Faraday.new(url: 'https://chat.stream-io-api.com'))
```

Comment:
would be nice to set a bogus one first and see the error then correct it

Reply:
✅ , let's see if CI succeeds